ProjectLighthouse/Levels/Slot.cs

```diff
@@ -287,9 +295,18 @@ public string Serialize
     )
     {
         if (this.Type == SlotType.Developer) return this.SerializeDevSlot();
+        AdvSlot?[]? adventureSlots = this.adventureSlots;
+        string? advSlotsSerialized = null;
+        if (this.isAdventurePlanet && this.adventureSlots != null) {
+            advSlotsSerialized = "";
+            foreach (var slot in adventureSlots!)
+            {
+                advSlotsSerialized += slot!.Serialize();
+            }
+        }
```

Comment:
```suggestion
```
The sublevels don't need to be serialized. Here's an example serialization from a packet capture I found: https://pastebin.com/yKLvF9hb